index.html

```diff
@@ -1,203 +1,294 @@
-                    </ul>
-                  </div>
-                  <div class="programming-sec">
-                    <ul class="outer-item">
-                      <li class="programming"><img src="image/css.png" alt="CSS"></li>
-                      <li>
-                        <p>CSS</p>
-                      </li>
-                    </ul>
-                  </div>
-                  </div>
-                    <br>
-                      <div class="content-sec">
-                      <label>Frameworks</label>
-                      <img src="image/icon-right.png" alt="icon" class="icon-right">
+            <div class="programming-sec">
+              <ul class="outer-item">
+                <li class="programming">
+                  <img src="image/html.png" alt="HTML" />
+                </li>
+                <li>
+                  <p>HTML</p>
+                </li>
+              </ul>
+            </div>
+            <div class="programming-sec">
+              <ul class="outer-item">
+                <li class="programming">
+                  <img src="image/css.png" alt="CSS" />
+                </li>
+                <li>
+                  <p>CSS</p>
+                </li>
+              </ul>
+            </div>
+          </div>
+          <br />
+          <div class="content-sec">
+            <label>Frameworks</label>
+            <img src="image/icon-right.png" alt="icon" class="icon-right" />
+          </div>
+          <div class="underline"></div>
+          <div class="content-sec">
+            <label>Skills</label>
+            <img src="image/icon-right.png" alt="icon" class="icon-right" />
+          </div>
+          <div class="underline"></div>
+        </div>
+      </section>
+    </main>
 
-                  </div>
-                  <div class="underline"></div>
-                  <div class="content-sec">
-                      <label>Skills</label>
-                          <img src="image/icon-right.png" alt="icon" class="icon-right">
-                        </div>
-                  <div class="underline"></div>
-                </div>
-              </section>
-        </main>
+    <footer class="footer">
+      <div class="contact-content">
+        <h2 class="contact-header">Contant Us</h2>
+        <p class="contact-paragraph">
+          If you have an application you are interested in developing, a feature
+          that you need built or a project that needs coding. I’d love to help
+          with it
+        </p>
+      </div>
+      <form action="https://formspree.io/f/xjvlllyy" method="POST">
+        <input
+          name="name"
+          id="user-name"
+          type="text"
+          placeholder="Name"
+          maxlength="30"
+          required
+        />
 
-</body>
+        <input
+          name="email"
+          id="user-email"
+          type="email"
+          placeholder="name@gmail.com"
+          required
+        />
 
+        <textarea
```

Comment:
- [ ] The form will send even if the user didn't add a text message, please fix this you can remove the text inside the ```textarea``` (Write your message here) and replace it using a placeholder
- [ ] Please try to use a placeholder for the ```textarea```, it's a requirement